lib/common/ipc_client.c

```diff
@@ -1266,7 +1266,7 @@ crm_ipc_send(crm_ipc_t *client, const xmlNode *message,
 
     iov_buffer = g_string_sized_new(1024);
     pcmk__xml_string(message, 0, iov_buffer, 0);
-    rc = pcmk__ipc_prepare_iov(id, iov_buffer, &iov, &bytes);
+    rc = pcmk__ipc_prepare_iov(id, iov_buffer, 0, &iov, &bytes);
 
     if (rc != pcmk_rc_ok) {
```

Comment:
This needs to also handle `pcmk_rc_ipc_more`. That's basically Coverity's complaint. `iov` is leaked in that case.

Reply:
Hm, yeah, this is actually coming in the next PR for non-event IPC.  Anything I do here is just going to be temporary until that gets finished (pretty close), reviewed, and merged.  I can add something like this:

```
if (rc != pcmk_rc_ok) {
    /* FIXME: Remove this when regular IPC supports split up messages */
    if (rc == pcmk_rc_ipc_more) {
        pcmk_free_ipc_event(iov);
    }

    ...
}
````

Reply:
I think this avoids the leak, but would we want to treat `pcmk_rc_ipc_more` as success instead?